docroot/themes/custom/twentyeighteen/sass/base/_base.scss

```diff
@@ -35,80 +37,74 @@ body {
     }
   }
 }
+
+hr {
+  color: $teal;
+  border-width: 2pt;
+  border-style: solid;
+  margin: 2em 0;
+}
+
 // Base Headings
 //
 // Markup: headlines.twig
 //
 // Style guide: base.headlines
-h1 {
-  color: $sienna;
-  font-weight: 700;
-  font-family: $heading_font;
-}
-
+h1,
 h2,
 h3,
 h4,
 h5,
 h6 {
-  color: $heading;
+  font-family: 'Montserrat', Helvetica, Arial, sans-serif;
+  text-rendering: optimizeLegibility;
   font-weight: 700;
-  font-family: $heading_font;
-  padding: 0 0 0.3em;
-  margin: 0;
+  color: $light-teal;
 }
 
 h1 {
+  color: $sienna;
   font-size: 2em;
-  line-height: 165%;
 }
 
 h2 {
+  color: $teal;
   font-size: 1.5em;
-  line-height: 145%;
 }
 
 h3 {
+  color: $teal;
   font-size: 1.25em;
-  line-height: 145%;
 }
 
 h4 {
-  font-size: 1.15em;
-  line-height: 145%;
+  font-size: 1.125em;
 }
 
-h5,
-h6 {
-  font-size: 1.05em;
-  line-height: 145%;
+h5 {
+  font-size: 1em;
 }
 
-body .menu-item {
-  padding-top: 0;
-  margin: 0;
+h6 {
+  font-size: 0.875em;
+  letter-spacing: 0.02em; /* Increases readibility at this smaller type size */
 }
+
 // Base Text
 //
 // Markup: text.twig
 //
 // Style guide: base.text
+
 a,
-a:link,
 a:visited {
+  color: $teal;
   text-decoration: none;
-  color: $link;
-  cursor: inherit;
-
-  &:hover {
-    color: $sienna;
```

Comment:
We shouldn't be deleting the hover color, underline or cursor as it's bad practice. If Karl doesn't have a hover color in his CSS, let's keep it as Sienna